elements/rh-sass/functions/_functions.scss

```diff
@@ -127,3 +127,11 @@
     }
     @return $result;
 }
+
+/// RH Var
+/// Uses the name created by rh-define-vars() to retrieve the local var
+/// @requires: @mixin rh-define-vars()
+/// @param {String} $this-local-var - Variable name to be used
+@function rh-var($this-local-var) {
```

Comment:
The $fallback should be handled with the `rh-define-vars()` mixin—which sets all the Vars including the fallback. I added the comment `@requires: @mixin rh-define-vars()`

The point of this function is to make calling the localized CSS Var easier. The localized CSS Var simply points to the CSS Vars stack with the fallback (in `rh-define-vars()`).

Reply:
Yes that's a good point but it might have more reuse as a function if we abstract it a bit.  For example if we rewrite the function like this:

```scss
@function rh-var($var-name, $namespace: local, $fallback: null) {
     @if ($fallback) {
         @return var(--rh-#{$namespace}-#{$var-name}, #{$fallback});
    } @else {
         @return var(--rh-#{$namespace}-#{$var-name});
    }
}
```

Now this becomes a multi-use function that can accept a different namespace and optional fallback (which can be either a color or another variable name).  Just an idea to make this really great function helpful for different use-cases.

Reply:
Good thought! You in next week (like Wednesday)? Let's work this out along with the map idea.

Reply:
Perfect - I'm in the office on Wednesdays so that's works great.

Reply:
Proposed mixin: https://www.sassmeister.com/gist/ceb6de36877eb49c69be87c29b5a1ba2